spec/features/successful_signin_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe ChitterApp do
+  feature 'Successfully signin' do
+    scenario 'User can enter their details and gains access to the app' do
+      User.new_user('littletrout', 'password1', 'littletrout@notadomain.com')
+      visit '/'
+      click_button 'Sign in'
+      fill_in 'username', with: 'littletrout'
+      fill_in 'password', with: 'password1'
+      click_button 'Sign in'
```

Comment:
You could use a web helper for this block, so you can keep your code dry and be able to reuse the test set up in any of your feature tests. It will also be easier to edit should you want to change the functionality of your program